src/tests/test_kube_api.py

```diff
@@ -406,15 +406,32 @@ def _late_binding_install(
     ) -> None:
         cls._bind_all(cluster_deployment, agents)
         cls._set_agent_cluster_install_machine_cidr(agent_cluster_install, nodes)
+        api_vip, _ = cls._get_vips(nodes)
 
         if len(nodes) == 1:
             cls.set_single_node_ip(cluster_deployment, nodes)
+            api_vip = get_ip_for_single_node(cluster_deployment, nodes.is_ipv4)
+
+        """
+        Add the API VIP DNS record to the assisted-service network
+        here so it can resolve by the time the host(s) start reclaim
+        """
```

Comment:
```suggestion
        # Add the API VIP DNS record to the assisted-service network
        # here so it can resolve by the time the host(s) start reclaim
```

Reply:
Modified, thank you!